packages/blade/src/components/Typography/Code/Code.tsx

```diff
@@ -63,6 +96,24 @@ const CodeContainer = styled(BaseBox)<CodeContainerProps>((props) => {
   };
 });
 
+const getCodeColor = ({
+  isHighlighted,
+  color,
+}: Pick<CodeProps, 'isHighlighted' | 'color'>): CodeProps['color'] => {
+  if (isHighlighted) {
+    if (color) {
+      throw new Error('[Blade: Code]: `color` prop cannot be used without `isHighlighted={false}`');
```

Comment:
This error sounds negated, can we simplify to: 

```suggestion
      throw new Error('[Blade: Code]: `color` prop cannot be used when `isHighlighted` is true');
```

Reply:
so `isHighlighted` is true by default so people might think "arey?? I didn't make `isHighlighted` true then why is this error coming"

Reply:
Ohh i see, okay.